bokeh/io.py

```diff
@@ -609,3 +610,31 @@ def _remove_roots(subplots):
     for sub in subplots:
         if sub in doc.roots:
             doc.remove_root(sub)
+
+def server_cell(server, script):
```

Comment:
I think `show_app` should be the `_show_notebook_app_with_state` outlined below, probably